### ISSUE TITLE
[PlayerDialogAdditions] "Do you know this person" option is broken

### ISSUE THREAD

**Issue by Factorionaut** [piepieonline/ShadowsOfDoubtMods#6]
I get this error when selecting "Do you know this person?" option. Clicking on the portraits does nothing. This is on experimental build 37.09, mods and BepInEx installed through Thunderstore.

```[Error  :Il2CppInterop] During invoking native->managed trampoline
Exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at DialogAdditions.PhotoSelectButtonController_OnLeftClick.DoYouKnowThisPersonAdditions(Human speaker, Human askTarget, List`1 askTargetKeys) in D:\Game Modding\ShadowsOfDoubt\_Mods\ShadowsOfDoubtMods\DialogAdditions\PhotoSelectOverrides.cs:line 257
   at DialogAdditions.PhotoSelectButtonController_OnLeftClick.Prefix(PhotoSelectButtonController __instance) in D:\Game Modding\ShadowsOfDoubt\_Mods\ShadowsOfDoubtMods\DialogAdditions\PhotoSelectOverrides.cs:line 34
   at DMD<PhotoSelectButtonController::OnLeftClick>(PhotoSelectButtonController this)
   at (il2cpp -> managed) OnLeftClick(IntPtr , Il2CppMethodInfo* )

**Comment by piepieonline** [comment]
Have you got procedural murders turned off?

**Comment by Factorionaut** [comment]
> Have you got procedural murders turned off?

Yes.

**Comment by piepieonline** [comment]
Fixed in v0.11.4
